java-service.gitlab-ci.yml

```diff
@@ -67,51 +67,34 @@ push_bundle:
     - npm ci
     - 'grunt bundle --packageVersion=$PACKAGE_VERSION --packageName=$CI_PROJECT_NAME'
 
-deploy_cit:
+.deploy_template: &deploy_definiton
   image: briangweber/node:8
-  stage: validate
   only:
     - master
   except:
     - schedules
-  environment:
-    name: cit
   script:
     - echo "@cimpress-technology:registry = https://artifactory.cimpress.io/artifactory/api/npm/npm-release-local/" >> ~/.npmrc
     - echo "//artifactory.cimpress.io/artifactory/api/npm/npm-release-local/:_authToken=${CT_ARTIFACTORY_NPM_TOKEN}" >> ~/.npmrc
     - npm i -g grunt
     - npm ci
-    - 'grunt deploy --environment=cit --packageVersion=$PACKAGE_VERSION --packageName=$CI_PROJECT_NAME'
+    - 'grunt deploy --environment=$CI_ENVIRONMENT_NAME --packageVersion=$PACKAGE_VERSION --packageName=$CI_PROJECT_NAME'
+
+deploy_cit:
```

Comment:
Do we maybe want to nuke this while we're in here?

Reply:
sure :)